solutions/USACO dec18-bucketlist.mdx

```diff
@@ -41,3 +41,42 @@ file_out.close()
 </PySection>
 
 </LanguageSection>
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>
```

Comment:
don't you need cstdio for freopen?

Reply:
iostream has freopen too

Reply:
iostream also has min/max too

Reply:
ok so ignore my comments above; add `#include <cstdio>`

Reply:
address this

Reply:
done i removed algorithm